client/src/styles/_common.scss

```
@@ -0,0 +1,152 @@
+@import './_variables';
+
+@mixin size($width, $height) {
+  width: $width;
+  height: $height;
+}
+
+@mixin position($position, $top, $right) {
+  position: $position;
+  top: $top;
+  right: $right;
+}
+
+@mixin font_style($font-weight, $font-size, $line-height) {
+  font-weight: $font-weight;
+  font-size: $font-size;
+  line-height: $line-height;
+}
+
+/* item 상태별 style */
+
+.default {
+  background-color: $white-color;
+  box-shadow: 0px 1px 30px rgba(224, 224, 224, 0.3);
+
+  .close-icon {
+    path {
+      fill: $gray3-color;
+    }
+  }
+}
+
+.drag {
+  background-color: rgba(255, 255, 255, 0.226);
+  box-shadow: 0px 0px 4px rgba(204, 204, 204, 0.5),
+    0px 2px 4px rgba(0, 0, 0, 0.25);
+  backdrop-filter: blur(4px);
+
+  .close-icon {
+    path {
+      fill: $gray3-color;
+    }
+  }
+}
+
+.place {
+  background: $white-color;
+  opacity: 0.4;
+  border: 1px solid #0075de;
+  box-shadow: 0px 1px 30px rgba(224, 224, 224, 0.3);
+
+  .close-icon {
+    path {
+      fill: $gray3-color;
+    }
+  }
+}
+
+.delete {
+  background: $white-red-color;
+  border: 1px solid $red-color;
+  box-shadow: 0px 1px 30px rgba(224, 224, 224, 0.3);
+
+  .close-icon {
+    path {
+      fill: $red-color;
+    }
+  }
+}
+
+.active {
+  border: 1px solid $blue-color;
+}
+
+.deactivate {
+  border: 1px solid $blue-color;
+
+  .item-title,
+  .item-content {
+    color: $gray3-color;
+  }
+
+  input {
+    width: 100%;
+    border: none;
+  }
+
+  input:focus {
+    outline: none;
+  }
+}
+
+/* 버튼 상태별 style */
+
+%default_button {
+  width: 134px;
+  padding: 10px;
+  border-radius: 6px;
+}
+
+.active-btns {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  button {
+    @extend %default_button;
+  }
+
+  .normal-btn {
+    margin-right: 8px;
+  }
+}
+
+.accent-btn {
+  background-color: $blue-color;
+  color: $white-color;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.accent-btn:hover {
+  background-color: $darkblue-color;
+}
+
+.accent-btn[disabled='false']:hover {
+  background-color: $darkblue-color;
+}
+
+.accent-btn[disabled='false'] {
+  background-color: $blue-color;
+}
+
+.accent-btn[disabled='true'] {
+  background-color: $lightblue-color;
+  color: rgba(255, 255, 255, 0.4);
+}
```

scss의 장점을 조금 더 사용해 볼 수 있을 것 같네요!  

```suggestion
.accent-btn {
  background-color: $blue-color;
  color: $white-color;
  font-weight: 700;
  font-size: 14px;
  &:hover {
    background-color: $darkblue-color;
  }
  [disabled='true'] {
    background-color: $lightblue-color;
    color: rgba(255, 255, 255, 0.4);
  }
}
```